app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -11,34 +11,31 @@ import { iconHTML } from "discourse-common/lib/icon-library";
 import discourseComputed from "discourse-common/utils/decorators";
 import I18n from "discourse-i18n";
 
-export default Controller.extend({
-  queryParams: ["q"],
-  q: null,
-
-  router: service(),
-  application: controller(),
-  user: controller(),
-  loading: false,
-  loadingMore: false,
-  permissionDenied: false,
-  inSearchMode: notEmpty("q"),
-  noContent: equal("model.bookmarks.length", 0),
-
-  bulkSelectHelper: null,
-
-  init() {
-    this._super(...arguments);
-    this.bulkSelectHelper = new BulkSelectHelper(this);
-  },
-
-  searchTerm: computed("q", {
-    get() {
-      return this.q;
-    },
-    set(key, value) {
-      return value;
-    },
-  }),
+export default class UserActivityBookmarksController extends Controller {
+  @service router;
+  @controller application;
+  @controller user;
+
+  queryParams = ["q"];
+
+  q = null;
+  loading = false;
+  loadingMore = false;
+  permissionDenied = false;
+
+  bulkSelectHelper = new BulkSelectHelper(this);
+
+  @notEmpty("q") inSearchMode;
+  @equal("model.bookmarks.length", 0) noContent;
+
+  @computed("q")
+  get searchTerm() {
+    return this.q;
+  }
+
+  set searchTerm(value) {
```

Comment:
I know, back compat and all, but is there anything that is setting this? 😃

Reply:
Yeah I dunno, and changing it is not a risk I wanna take for this PR 😅